### DISCUSSION TITLE
Single connection mode use case

### DISCUSSION THREAD

**Discussion by ashwanipahal** [enisdenjo/graphql-sse#76] [Q&A] [answered]
Hi @enisdenjo I am using single connection mode,  I am able to make connection only in 5 tabs and getting status 202 Accepted but if i open another tab, request status stays pending until i close other opened tab. Can you please help me wether it is default behaviour or i am missing something.
<img width="924" alt="Screenshot 2023-08-24 at 11 04 43 PM" src="https://github.com/enisdenjo/graphql-sse/assets/13705046/f795fe85-6cdb-461f-acea-e87d96e4017f">

**Comment by enisdenjo** [comment, marked as answer]
Hey there, SSE has a known limitation in browsers of 6 active connections per origin when served over HTTP/1.1. There's not much you can do here except use an HTTP/2 enabled server.

> Warning: When not used over HTTP/2, SSE suffers from a limitation to the maximum number of open connections, which can be especially painful when opening multiple tabs, as the limit is per browser and is set to a very low number (6). The issue has been marked as "Won't fix" in [Chrome](https://crbug.com/275955) and [Firefox](https://bugzil.la/906896). This limit is per browser + domain, which means that you can open 6 SSE connections across all of the tabs to www.example1.com and another 6 SSE connections to www.example2.com (per [Stackoverflow](https://stackoverflow.com/questions/5195452/websockets-vs-server-sent-events-eventsource/5326159)). When using HTTP/2, the maximum number of simultaneous HTTP streams is negotiated between the server and the client (defaults to 100).

[_Using server-sent events on MDN_](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events#receiving_events_from_the_server)
